nimare/decode/continuous.py

```diff
@@ -225,7 +235,59 @@ def _run_fit(self, i_feature, feature, dataset):
             return_type="array",
         )
 
-        return feature_data, i_feature
+        return feature, feature_data
+
+    def load_imgs(self, features_imgs, mask=None):
+        """Load pregenerated maps from disk.
+
+        .. versionadded:: 0.1.0
+
+        Parameters
+        ----------
+        features_imgs : :obj:`dict`, or str
+            Dictionary with feature names as keys and paths to images as values.
+            If a string is provided, it is assumed to be a path to a folder with NIfTI images,
+            where the file's name (without the extension .nii.gz) will be considered as the
+            feature name by the decoder.
+        mask : str, :class:`nibabel.nifti1.Nifti1Image`, or any nilearn Masker
+            Mask to apply to pre-generated maps.
+
+        Attributes
+        ----------
+        results_ : :obj:`~nimare.results.MetaResult`
+            MetaResult with meta-analytic maps and masker added.
+        """
+        if isinstance(features_imgs, dict):
+            feature_imgs_dict = features_imgs
+        elif isinstance(features_imgs, str):
+            img_paths = sorted(glob(os.path.join(features_imgs, "*.nii*")))
+            img_names = [os.path.basename(img).split(os.extsep)[0] for img in img_paths]
+            feature_imgs_dict = dict(zip(img_names, img_paths))
+        else:
+            raise ValueError(
+                f'"feature_imgs" must be a dictionary or a string, not a {type(features_imgs)}.'
+            )
+
+        # Replace attributes of initialized class self with Nones, so that default values are not
+        # confused with the parameters used before to generate the maps that are read from disk.
+        for attr in self.__dict__:
+            setattr(self, attr, None)
+
+        if mask is not None:
+            mask = get_masker(mask)
+        else:
+            raise ValueError("A mask must be provided.")
+        self.masker = mask
+
+        # Load pre-generated maps
+        features, images = ([], [])
+        for feature, img_path in feature_imgs_dict.items():
+            img = nib.load(img_path)
+            features.append(feature)
+            images.append(np.squeeze(self.masker.transform(img)))
+
+        maps = {feature: image for feature, image in zip(features, images)}
+        self.results_ = MetaResult(self, mask=self.masker, maps=maps)
 
     def transform(self, img):
```

Comment:
My only question is whether we should allow this to be configurable and allow only the data frame the returned?

This would prevent a lot of copies of the `MetaResult` object and images if somebody is doing multiple decodings and doesn't want to save the images.

Also-- what happens when you save a `MetaResult` object? Do the images get saved or also the `tables`?